contracts/account/modules/ERC7579DelayedExecutor.sol

```diff
@@ -24,22 +24,24 @@ import {ERC7579Executor} from "./ERC7579Executor.sol";
  * executable. If an operation is not executed within the expiration period, it becomes `Expired`
  * and can't be executed. Expired operations must be rescheduled with a different salt.
  *
- * ==== Delay Management
+ * #### Delay Management
  *
  * Accounts can set their own delay periods during installation or via {setDelay}.
  * The delay period is enforced even between installas and uninstalls to prevent
  * immediate downgrades. When setting a new delay period, the new delay takes effect
  * after a transition period defined by the current delay or {minSetback}, whichever
  * is longer.
  *
- * ==== Authorization
+ * #### Authorization
  *
  * Authorization for scheduling and canceling operations is controlled through the {_validateSchedule}
  * and {_validateCancel} functions. These functions can be overridden to implement custom
  * authorization logic, such as requiring specific signers or roles.
  *
- * TIP: Use {_scheduleAt} to schedule operations at a specific points in time. This is
+ * <Callout>
```

Comment:
I feel we are loosing information by merging notes and tips in a same callout type. Could we keep `Tip:` prefix or something like that (?):
```suggestion
 * <Callout icon="bulb">
```

Reply:
Thank you, the UI is not super different but I would use idea callouts for tips, and default callouts as you did for notes.